app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/ProfileLoginFragmentTests.kt

```diff
@@ -503,8 +545,6 @@ class ProfileLoginFragmentTests {
             endingRequest
         }
         onView(withId(R.id.id_btn_login_button)).perform(click())
-        //Notify that the firstConnection request was successfully performed
-        endingRequestFirstConnection.postValue(false)
         //Notify that the getUserAInformation request was successfully performed
         endingRequest.postValue(false)
         Thread.sleep(2000)
```

Comment:
Isn't it a lot of time for a test?

Reply:
Actually, since the operation is asynchronous, it's more safe to put 2 second :)